data.json

```diff
@@ -777,5 +777,21 @@
     "installCount": 228,
     "lastUpdate": "20 Mar 2021",
     "stars": 13
+  },
+  {
+    "name": "Old School GitHub",
+    "source": "https://github.com/daattali/oldschool-github-extension",
+    "tags": [
+      "github",
+      "chrome-extension",
+      "firefox-addon",
+      "firefox-extension",
+      "chrome-extensions"
```

Comment:
Tags will appear as filters on https://stefanbuck.com/awesome-browser-extensions-for-github. My suggestion is to remove all of them and use just `theme`

Reply:
@stefanbuck Done!